### PR TITLE
Evanbranch

### PR DESCRIPTION
**Pull request by evanrich2404**
I am not quite sure if I missed anything, I also am not entirely sure if it is going to work as expected but I wanted to put this out anyways since most of the work has been done. Please review and tell me if you see anything I can't. I will keep trying to figure out how to test it on my local machine and also try out the sandbox

### CODE REVIEW

**Review by Oggal — COMMENTED** [review]
Haven't tested it on my machine yet, but figured we could merge tomorrow once tested

File: models/engine/db_storage.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 """This module defines a class to manage sql storage for hbnb clone"""
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from models.base import Base
+from sqlalchemy.orm import relationship
```
[the remark this answers is not in the document]
Comment (evanrich2404) [reply models/engine/db_storage.py:5]: I believe I temporarily used it but forgot to remove the import when I figured out what I was doing

File: models/city.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/python3
 """ City Module for HBNB project """
-from models.base_model import BaseModel
+from models.base_model import BaseModel, Base
+from sqlalchemy import Column, String, ForeignKey, Integer
 
 
 class City(BaseModel):
```
[the remark this answers is not in the document]
Comment (evanrich2404) [reply models/city.py]: I believe so? Not sure I understand this question

File: models/state.py [hunk 3]
```diff
@@ -1,8 +1,27 @@
 #!/usr/bin/python3
 """ State Module for HBNB project """
-from models.base_model import BaseModel
+from models.base_model import BaseModel, Base
+from models.city import City
+from sqlalchemy import Column, String
+import os
 
 
 class State(BaseModel):
```
[the remark this answers is not in the document]
Comment (evanrich2404) [reply models/state.py]: I now understand what you were talking about

**Pull request merged**